shared/middleware/helpers/auth_helpers.rb

```diff
@@ -30,13 +30,12 @@ def under_13?
   age < 13
 end
 
-# Returns the sharing_disabled property of a user with a given user_id
+# Returns the sharing_disabled property of a user with a given user_id,
+# always defaulting sharing to enabled.
```

Comment:
as shown by some current tests, sometimes files won't have an owner, and in that case, we want to default to sharing_disabled = false, as well as when no property is set.